config/consts.js

```diff
@@ -3,14 +3,19 @@
  */
 
 const APP_INDEX_HTML = 'index.html';
-const APP_INDEX_JS = 'index.js';
+const APP_INDEX_JS = 'app.js';
 
 const MEDIA_FILE_REGEX = /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2)(\?.*)?$/;
 const MEDIA_URL_REGEX = /\.(mp4|webm|wav|mp3|m4a|aac|oga)(\?.*)?$/;
 
+const AUTH0_CLIENT_ID = 'KvwsETaUxuXVjW2cmz2LbdqXQBdYs6wH';
+const AUTH0_DOMAIN = 'loom.auth0.com';
+
 module.exports = {
```

Comment:
`module.exports` is a NodeJS thing. in ES6, we'll be using the `export` keyword. look at this for reference:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export
https://github.com/eslint/espree/pull/43